### ISSUE TITLE
UX Logic

### ISSUE THREAD

**Issue by Hamdan85** [grishkovelli/vue-audio-recorder#26]
To make this awesome package even more usefull... would be possible to make it a single audio recorder like a whatsapp audio recorder for example... release to send

**Comment by grishkovelli** [comment]
What do you mean?

**Comment by Abdulaziz-almoshen** [comment]
I agree with Hamdan, the UX is good but too big and it would be great to make it like an icon to record then review the audio like WhatsApp
